src/test/java/com/wavesplatform/wavesj/json/ser/AssetDetailsSerTest.java

```diff
@@ -0,0 +1,30 @@
+package com.wavesplatform.wavesj.json.ser;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.wavesplatform.wavesj.AssetDetails;
+import com.wavesplatform.wavesj.json.WavesJsonMapper;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.io.StringWriter;
+
+import static org.junit.Assert.assertEquals;
+
+public class AssetDetailsSerTest {
```

Comment:
I wonder why **TransactionSerTest** was not used as a parent here and you did a copy/past ?
Do you have some issues with this base class ?